napalm_eos/eos.py

```diff
@@ -1378,3 +1377,56 @@ def _append(bgp_dict, peer_info):
             _append(bgp_detail_info, peer_info)
 
         return bgp_detail_info
+
+    def get_optics(self):
+
+        command = ['show interfaces transceiver']
+
+        output = (
+            self.device.run_commands(
+                command, encoding='json')[0]['interfaces'])
+
+        # Formatting data into return data structure
+        optics_detail = []
```

Comment:
It's failing now because the output is a list instead of dict as expected in the test base: https://github.com/napalm-automation/napalm-base/blob/develop/napalm_base/test/base.py#L476